internal/services/cockpit/alert_manager.go

```diff
@@ -0,0 +1,256 @@
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	projectID := d.Get("project_id").(string)
+	emails := d.Get("emails").([]interface{})
+
+	_, err = api.EnableAlertManager(&cockpit.RegionalAPIEnableAlertManagerRequest{
+		Region:    region,
+		ProjectID: projectID,
+	})
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	_, err = api.EnableManagedAlerts(&cockpit.RegionalAPIEnableManagedAlertsRequest{
+		Region:    region,
+		ProjectID: projectID,
+	})
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	if len(emails) > 0 {
+		for _, email := range emails {
+			emailStr, ok := email.(string)
+			if !ok {
+				return diag.FromErr(errors.New("invalid email format"))
+			}
+			emailCP := &cockpit.ContactPointEmail{
+				To: emailStr,
+			}
+			_, err := api.CreateContactPoint(&cockpit.RegionalAPICreateContactPointRequest{
+				ProjectID: projectID,
+				Email:     emailCP,
+				Region:    region,
+			}, scw.WithContext(ctx))
+			if err != nil {
+				return diag.FromErr(err)
+			}
+		}
+	}
+
+	d.SetId(ResourceCockpitAlertManagerID(region, projectID))
+	return ResourceCockpitAlertManagerRead(ctx, d, meta)
+}
+
+func ResourceCockpitAlertManagerRead(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	api, region, err := cockpitAPIWithRegion(d, meta)
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	projectID := d.Get("project_id").(string)
+
+	alertManager, err := api.GetAlertManager(&cockpit.RegionalAPIGetAlertManagerRequest{
+		Region:    region,
+		ProjectID: projectID,
+	}, scw.WithContext(ctx))
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	_ = d.Set("enable_managed_alerts", alertManager.ManagedAlertsEnabled)
+	_ = d.Set("region", alertManager.Region)
+	_ = d.Set("alert_manager_url", alertManager.AlertManagerURL)
+
+	contactPoints, err := api.ListContactPoints(&cockpit.RegionalAPIListContactPointsRequest{
+		Region:    region,
+		ProjectID: projectID,
+	}, scw.WithContext(ctx))
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	var emails []string
+	for _, cp := range contactPoints.ContactPoints {
+		if cp.Email != nil {
+			emails = append(emails, cp.Email.To)
+		}
+	}
+	_ = d.Set("emails", emails)
+	return nil
+}
+
+func ResourceCockpitAlertManagerUpdate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	api, region, err := cockpitAPIWithRegion(d, meta)
+	if err != nil {
+		return diag.FromErr(err)
+	}
+	projectID := d.Get("project_id").(string)
+
+	if d.HasChange("enable_managed_alerts") {
+		enable := d.Get("enable_managed_alerts").(bool)
+		if enable {
+			_, err = api.EnableManagedAlerts(&cockpit.RegionalAPIEnableManagedAlertsRequest{
+				Region:    region,
+				ProjectID: projectID,
+			})
+		} else {
+			ar, err := api.DisableManagedAlerts(&cockpit.RegionalAPIDisableManagedAlertsRequest{
+				Region:    region,
+				ProjectID: projectID,
+			}, scw.WithContext(ctx))
+			_ = ar
+			_ = err
+		}
+
+		if err != nil {
+			return diag.FromErr(err)
+		}
+	}
```

Comment:
Direct error handling and remove unused `ar` variable
```suggestion
	if d.HasChange("enable_managed_alerts") {
		enable := d.Get("enable_managed_alerts").(bool)
		if enable {
			_, err = api.EnableManagedAlerts(&cockpit.RegionalAPIEnableManagedAlertsRequest{
				Region:    region,
				ProjectID: projectID,
			}, scw.WithContext(ctx))
			if err != nil {
				return diag.FromErr(err)
			}
		} else {
			_, err = api.DisableManagedAlerts(&cockpit.RegionalAPIDisableManagedAlertsRequest{
				Region:    region,
				ProjectID: projectID,
			}, scw.WithContext(ctx))
			if err != nil {
				return diag.FromErr(err)
			}
		}
	}

```